licenses/LICENSE.testng.al20.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Don't know if this is needed for a developement-time dependency (we don't package it into Armeria). Otherwise, our web licenses file is also missing a lot of development-time dependencies.

Reply:
@anuraaga 
I think it depends on who we define as the "receiver" of Armeria. 
- If we define the receiver as the person who clones this repository, then we should notify them of the test library license. 
- Or if we define the receiver as the person who downloads Armeria release, then we don't need to.

Reply:
So what do you think? 
Since we don't notify the licenses of front-end dev-dependencies, we don't need to add testing lib license either as @anuraaga said?
But @ikhoon said there are some licenses we are notifying, such as `mockito` and `junit`.

Reply:
I'd recommend removing the other current test dependency licenses too. The main reason for license notification IMO is so users can easily confirm whether there code is license-compatible with Armeria. This doesn't apply to test code, only published code.

Tests don't have any compatibility issue for users, only for this repository itself which we can confirm when adding a new test dependency.

Reply:
> I'd recommend removing the other current test dependency licenses too.

+1